tests/ui.rs

```diff
@@ -338,6 +343,20 @@ fn main() -> Result<()> {
     if cfg!(unix) && target == host {
         ui(Mode::Pass, "tests/native-lib/pass", &target, WithoutDependencies, tmpdir.path())?;
         ui(Mode::Fail, "tests/native-lib/fail", &target, WithoutDependencies, tmpdir.path())?;
+        if cfg!(all(
+            any(target_arch = "x86", target_arch = "x86_64"),
+            target_os = "linux",
+            target_env = "gnu"
+        )) {
+            // No pass tests here currently.
```

Comment:
Uh, why did you do this?

If you thought I asked for this, then you misunderstood.
There should be a `tracing` folder inside the existing pass/fail folders.